ERCS/erc-7871.md

```diff
@@ -0,0 +1,139 @@
+---
+eip: 7871
+title: Wallet Signing API
+description: Adds a JSON-RPC method for requesting a signature from a wallet
```

Comment:
It would be nice if you could cram some details about how this proposal is different from `personal_sign` / `eth_sign` in here.